Source/Core/DoublyLinkedList.js

```diff
@@ -0,0 +1,105 @@
+/*global define*/
+define([
+        '../Core/defined',
+        '../Core/defineProperties'
+    ], function(
+        defined,
+        defineProperties) {
+    'use strict';
+
+    /**
+     * @private
+     */
+    function DoublyLinkedList() {
```

Comment:
Even though this is private, we should still properly document everything in this class so that contributors and maintainers know what's going on.